### PR TITLE
enables generation of a transaction protobuf that hasnt been signed

### LINKED ISSUE #101: CryptoTransferTransaction toProto() and toBytes() enforce validation of sigs
We have use cases were a transaction needs to be built but not signed until later on. The current implementation of `toProto()` and `toBytes()` enforces a validation that at least one signature is present.

Can we implement a `no validation` flag or alternative, or at least a `validateWithoutSignatureCheck`

### PR DESCRIPTION
**Pull request by QuestofIranon**
closes #101 

This introduces the ability to generate unsigned protobufs and byte arrays through the use of `unsigned` variants of `Validate`, `toProto`, and `toBytes`

### CODE REVIEW

File: src/main/java/com/hedera/hashgraph/sdk/account/AccountAddClaimTransaction.java [hunk 1]
```diff
@@ -19,7 +19,7 @@ public AccountAddClaimTransaction(Client client) {
         super(client);
     }
 
-    AccountAddClaimTransaction() {
+    public AccountAddClaimTransaction() {
```
Comment (abonander) [inline src/main/java/com/hedera/hashgraph/sdk/account/AccountAddClaimTransaction.java]: Why are all these constructors set to `public` here? These are for testing only.

Comment (QuestofIranon) [reply src/main/java/com/hedera/hashgraph/sdk/account/AccountAddClaimTransaction.java]: Making it possible to create an unsigned transaction without a client

**Comment by QuestofIranon** [comment]
@gregscullard Can you review this to see if it matches the functionality you were looking for?

**Comment by gregscullard** [comment]
I've added an example `TxGenerateSignSend.java` to illustrate how this would work, however I have a question.
When the transaction is created, a `nodeId` is specified. When the transaction is eventually sent to the network, it *must* be sent to the same node. E.g. you can't specify 0.0.3 and send to 0.0.4.

**Comment by lbaird** [comment]
Right. If you’re going to ask a node to do some work for you, then you have to pay that node. You can’t just pay someone else. So you have to include that node ID in the transaction you sign. And then you have to send that signed transaction to that node. Because, if you sent it to someone else, they wouldn’t get paid for their efforts in submitting it for you.


________________________________

**Comment by mehcode** [comment]
The SDK has code to ensure the node selection is matched to the node ID in the protobuf.

**Comment by gregscullard** [comment]
In which case I'm happy with the PR.

**Comment by QuestofIranon** [comment]
Updated it so it matches the new standards and can be merged into master.

 @gregscullard I accidentally wiped your example, I didn't realize it was actually committed to my branch when I did a local rebase from master and push. Can you re-add your example or do it as a separate pull request?

**Comment by gregscullard** [comment]
I'll do a separate PR, so I can test the changes at the same time against master ;)

**Pull request merged**